gloss/crt0.S

```diff
@@ -260,6 +260,9 @@ _skip_init:
   csrr a5, mstatus
   ori a5, a5, 0x200
   csrw mstatus, a5
+  csrwi vstart, 0
```

Comment:
We are less likely to encounter X-propagation-related failures if we actually initialize the `vtype` and `vl` CSRs.  Even though what you wrote is sufficient for executing whole-register loads and stores,we will save ourselves some pain if we just do something like `vsetivli x0, 0, e8,m1,ma,ta` here.

Reply:
Thanks!

Reply:
By the way, I'm using spaces between the `vtype` sub-fields, as agreed here: https://github.com/riscv/riscv-v-spec/pull/693

Reply:
ack